index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>M J portfolio</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <!-- mobile header -->
+    <header>
+      <nav>
+        <h1>My Logo</h1>
+        <div class="bars">
+          <div class="bar"></div>
+          <div class="bar"></div>
+          <div class="bar"></div>
+        </div>
+      </nav>
+    </header>
+    <!-- mobile main section -->
+    <main>
+      <section class="frontpage">
+        <h2>Hey There. I’m Ronald.</h2>
+        <h3>I'm a Software Developer</h3>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hesitate to contact me.
+        </p>
+      </section>
+      <footer>
+        <ul>
+          <li>
+            <img src="./images/github.svg" alt="github" />
+          </li>
+          <li>
+            <img src="./images/linkedin.svg" alt="linkedinicon" />
+          </li>
+          <li>
+            <img src="./images/evangelist.svg" alt="evangelisticon" />
+          </li>
+          <li>
+            <img src="./images/twitter.svg" alt="twittericon" />
+          </li>
+          <li>
+            <img src="./images/myspace.svg" alt="myspaceicon" />
+          </li>
```

Comment:
Each of the icons need to be inside an `a` tag. Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.